gov2/bedrock-runtime/README.md

```diff
@@ -62,14 +62,6 @@ functions within the same service.
 
 
 <!--custom.instructions.start-->
-#### Region configuration
-By default, examples are set to `us-east-1`. To specify a different region, use the `-region` flag as shown in this example:
-
-```
-go run ./hello -region=eu-central-1
-```
-
-Be aware that not all regions may support Bedrock and its models yet. Verify service availability for your region [here](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/). For available models per region, refer to the [Bedrock dashboard](https://console.aws.amazon.com/bedrock) in the AWS Management Console.
```

Comment:
Did you intend to remove this section about Regions?

Reply:
No. Added it back